index.d.ts

```diff
@@ -31,12 +32,16 @@ export class Client {
 }
 
 export class Frame {
-  constructor(command: string, headers?: {}, body?: string);
+  command: string;
+  body: string;
+  headers: ExtendedHeaders;
 
+  constructor(command: string, headers?: {}, body?: string);
```

Comment:
Since you are exposing `headers` as `ExtendedHeaders`, `headers` parameter in `constructor` should also be declared as `ExtendedHeaders`.